src/diem/testing/miniwallet/app/app.py

```diff
@@ -171,7 +188,33 @@ def _send_external_payment_txn(self, txn: Transaction) -> None:
         else:
             self._start_external_payment_txn(txn)
 
-    def _start_external_payment_txn(self, txn: Transaction) -> None:
+    def _start_external_payment_txn(self, txn: Transaction) -> None:  # noqa: C901
+        if identifier.diem_id.is_diem_id(str(txn.payee)):
+            reference_id = self._create_reference_id(txn)
+            try:
+                vasp_identifier = identifier.diem_id.get_vasp_identifier_from_diem_id(str(txn.payee))
+                domain_map = self.diem_client.get_diem_id_domain_map()
+                payee_onchain_address = domain_map.get(vasp_identifier)
```

Comment:
We need name this diem_id_address, because later in this func we got a payee_onchain_address from the response